charts/external-dns/templates/deployment.yaml

```
@@ -40,7 +40,7 @@ spec:
     {{- if not (quote .Values.automountServiceAccountToken | empty) }}
       automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
     {{- end }}
-      {{- with .Values.imagePullSecrets }}
+      {{- with (.Values.global.imagePullSecrets | default .Values.imagePullSecrets) }}
```

```suggestion
      {{- with (default .Values.global.imagePullSecrets .Values.imagePullSecrets) }}
```
The chart value should take precedence over the global value.